javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -11,39 +11,42 @@
 
 import java.io.Serializable;
 import java.util.*;
-import java.util.Set;
 import java.util.function.*;
 import java.util.stream.Collector;
 
 import static javaslang.collection.Collections.*;
 
 /**
- * Vector is the default Seq implementation. It provides the best performance in between Array (with constant time element access)
- * and List (with constant time element addition).
+ * Vector is the default Seq implementation that provides effectively constant time access to any element.
+ * Many other operations (e.g. `tail`, `drop`, `slice`) are also effectively constant.
  *
  * @param <T> Component type of the Vector.
- * @author Ruslan Sennov
+ * @author Ruslan Sennov, Pap Lőrinc
+ * @see BitMappedTrie
```

Comment:
I think this can be useful when someone's viewing the sources (I commented `BMT` thoroughly).
@danieldietrich?